content/articles/choosing-wagtail.md

```diff
@@ -0,0 +1,404 @@
+              
+            </td>
+        </tr>
+    </tfoot>
+</table>
+
+Django CMS and Wagtail are the two CMS frameworks that met all of our combined
+requirements. So, we compared how each against another set of developer-centric
+criteria:
+
+* Can the CMS __exist with other applications__ in a Django project?
+* How __flexible__ is the framework? Can the data model, and the CMS model be
+__extended__?
+* How __simple__ is it to implement, and to use?
+* Is the CMS UI device agnostic (__responsive__)?
+* Can we integrate a search engine?
+* What is the extend of __support and documentation__? How active is the
+__community__ around the CMS framework?
+* How easy is it to __learn to use__ the CMS framework?
+
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th style="width: 20%;">
+            Criteria
+            </th>
+            <th class="text-center" style="width: 40%;">
+            <img src="/img/assets/thumb-djcms.png" style="width: 150px" alt="Django CMS" title="Django CMS" aria-label="Django CMS" />
+            </th>
+            <th class="text-center" style="width: 40%;">
+            <img src="/img/assets/thumb-wagtail.png" style="width: 150px" alt="Wagtail" title="Wagtail" aria-label="Wagtail" />
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+            <b>Multiple applications in one project</b>
+            </td>
+            <td>
+            <small>CMS UI uses Django admin code, can’t separate admin and CMS UI. More of a standalone product.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Non-invasive Django Wagtail app. Admin for Wagtail is separate from Django admin. Can still manage applications code in the same project without using Wagtail CMS.</small>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            <b>Flexible and extendable model/CMS UI</b>
+            </td>
+            <td>
+            <small>Since it uses Django admin code, it is not easy to extend or be customized.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Very flexible and customizable. Data model is extendable, and can be shaped according to our requirements. Has more third party plugins in GitHub.</small>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            <b>Simplicity</b>
+            </td>
+            <td>
+            <small>UI design is complex from start, and because it is not easily customizable, it can’t be simplified.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Base CMS is simple, but functional. We can shape the CMS from scratch, according to our own requirements.</small>
```

Comment:
I might change but to and: "simple, and functional"

Reply:
Fixed